### ISSUE TITLE
When entering navigation, braille highlights should be on

### ISSUE THREAD

**Issue by NSoiffer** [NSoiffer/MathCATForPython#30]
Currently, when you use NVDA+ALT+m, you enter the MathML but the highlighting doesn't show up until you do a navigation command. It should be there from the start.
